src/frontend/web_application/src/components/MultidimensionalPi/components/Ratings/index.jsx

```diff
@@ -2,36 +2,41 @@ import React, { PropTypes } from 'react';
 import classnames from 'classnames';
 import './style.scss';
 
-const Rating = ({ name, level, piMax, className }) => {
+function classFor(element, mini) {
+  const classNames = [`m-pi-ratings__${element}`];
+  if (mini) { classNames.push(`m-pi-ratings--mini__${element}`); }
+
+  return classNames;
+}
+
+const Rating = ({ name, level, piMax, className, mini }) => {
   const width = (level / piMax) * 100;
   const style = { width: `${width}%` };
-  const ratingClassName = classnames(
-    'm-pi-ratings__item',
-    className,
-  );
 
   return (
-    <div className={ratingClassName}>
-      <div className="m-pi-ratings__item-name">
-        <span className="m-pi-ratings__item-name-label">{name}</span>
+    <div className={classnames(classFor('item', mini), className)}>
```

Comment:
well this is actually just fine of course :)

There is another way, and a bit more declarative : 

`className={classnames('m-pi-ratings__item`, { 'm-pi-ratings--mini__item': mini }, className)}`

it is not much more characters and it has the advantage to easily find a file containing the class (for example "m-pi-ratings--mini__item") when searching on the whole project in your IDE.